src/bridgesourcesink.py

```diff
@@ -79,7 +81,7 @@ def load_data_into_cells(self):
             # Name
             name_col = QLineEdit()
             name_col.setText(data.name)
-            name_col.returnPressed.connect(self.enable_buttons)
+            name_col.textChanged.connect(self.customChanged.emit)
```

Comment:
please note this usage is incorrect.
you are connecting here to a function rather than the actual signal.
you should remove the "emit" at the end.

this still works anyway because pyqt allows to objects to connect to regular python functions, so when you call "emit()" you are triggering that function..